.github/workflows/build.yml

```diff
@@ -595,6 +595,72 @@ jobs:
         with:
           name: contour-ubuntu2204-builddir
 
+  ubuntu_2004_packaging:
+    name: "Ubuntu 20.04: Create package"
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
```

Comment:
```suggestion
        uses: hendrikmuhs/ccache-action@v1.2.2
```

Reply:
Is this an in-place change that works out of the box or does it require more changes than simply adapting the number ?

Reply:
Works out of the box I hope.
Also actions supports `@latest` tag if I'm not mistaken.